frontend/src/components/NavigationButtons.tsx

```
@@ -0,0 +1,38 @@
+          onClick={onBack}
+          className="px-4 py-2.5 h-10 bg-gray-200 text-black rounded-full text-sm font-medium hover:bg-gray-300 transition-colors min-w-[92px]"
+        >
+          {t('diagnostic.buttons.back')}
+        </button>
+      )}
+
+      {step < 3 && (
+        <button
+          onClick={onNext}
+          className="px-4 py-2.5 h-10 bg-[#215E6B] text-white rounded-full text-sm font-medium hover:bg-[#1a4d57] transition-colors min-w-[92px]"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)